### ISSUE TITLE
Fails in repos without a master branch

### ISSUE THREAD

**Issue by echennells** [Azure/acr-build#2]
Newer github repos default branch is renamed main.  I tried putting in a BRANCH variable but it didn't seem to pick it up.

**Comment by johnazariah** [comment]
I ran into this issue as well

I just added 

```
          branch: main
```

into the `with:` section and it worked after

**Comment by ams0** [comment]
Indeed, we choose `master` as [default](https://github.com/Azure/acr-build/blob/b94de947eb63665ef2454844aee3ce2a1244d9d6/entrypoint.sh#L6). As everyone, I'm aware of the switch to `main` default branch name for new repos, but I wonder how many repos are out there still using `master`. It's a though decision to choose a default, and this is no different.
